src/components/FAQ.astro

```diff
@@ -0,0 +1,75 @@
+---
+import Question from "./Question";
+
+const questions: Array<{
+  question: string;
+  answer: string;
+}> = [
+  {
+    question: "Quis autem vel eum iure reprehenderit?",
+    answer:
+      "Sed ut perspiciatis unde omnis iste natus error sit voluptatem accusantium doloremque laudantium, totam rem aperiam, eaque ipsa quae ab illo inventore veritatis et quasi architecto.",
+  },
+  {
+    question: "Quis autem vel eum iure reprehenderit?",
+    answer:
+      "Sed ut perspiciatis unde omnis iste natus error sit voluptatem accusantium doloremque laudantium, totam rem aperiam, eaque ipsa quae ab illo inventore veritatis et quasi architecto.",
+  },
+  {
+    question: "Quis autem vel eum iure reprehenderit?",
+    answer:
+      "Sed ut perspiciatis unde omnis iste natus error sit voluptatem accusantium doloremque laudantium, totam rem aperiam, eaque ipsa quae ab illo inventore veritatis et quasi architecto.",
+  },
+  {
+    question: "Quis autem vel eum iure reprehenderit?",
+    answer:
+      "Sed ut perspiciatis unde omnis iste natus error sit voluptatem accusantium doloremque laudantium, totam rem aperiam, eaque ipsa quae ab illo inventore veritatis et quasi architecto.",
+  },
+  {
+    question: "Quis autem vel eum iure reprehenderit?",
+    answer:
+      "Sed ut perspiciatis unde omnis iste natus error sit voluptatem accusantium doloremque laudantium, totam rem aperiam, eaque ipsa quae ab illo inventore veritatis et quasi architecto.",
+  },
+  {
+    question: "Quis autem vel eum iure reprehenderit?",
+    answer:
+      "Sed ut perspiciatis unde omnis iste natus error sit voluptatem accusantium doloremque laudantium, totam rem aperiam, eaque ipsa quae ab illo inventore veritatis et quasi architecto.",
+  },
+  {
+    question: "Quis autem vel eum iure reprehenderit?",
+    answer:
+      "Sed ut perspiciatis unde omnis iste natus error sit voluptatem accusantium doloremque laudantium, totam rem aperiam, eaque ipsa quae ab illo inventore veritatis et quasi architecto.",
+  },
+  {
+    question: "Quis autem vel eum iure reprehenderit?",
+    answer:
+      "Sed ut perspiciatis unde omnis iste natus error sit voluptatem accusantium doloremque laudantium, totam rem aperiam, eaque ipsa quae ab illo inventore veritatis et quasi architecto.",
+  },
+  {
+    question: "Quis autem vel eum iure reprehenderit?",
+    answer:
+      "Sed ut perspiciatis unde omnis iste natus error sit voluptatem accusantium doloremque laudantium, totam rem aperiam, eaque ipsa quae ab illo inventore veritatis et quasi architecto.",
+  },
+  {
+    question: "Quis autem vel eum iure reprehenderit?",
+    answer:
+      "Sed ut perspiciatis unde omnis iste natus error sit voluptatem accusantium doloremque laudantium, totam rem aperiam, eaque ipsa quae ab illo inventore veritatis et quasi architecto.",
+  },
+];
+---
+
+<div class="container flex max-w-3xl flex-col items-center text-white">
+  <h1 class="mb-10 text-center font-display text-6xl font-bold">
```

Comment:
Wait, please don't use h1 for anything other than the page title. This is very bad for accessibility.

Reply:
Will fix!